templates/zerver/app/left_sidebar.html

```diff
@@ -60,8 +60,12 @@
         </ul>
         <div id="streams_list" class="zoom-out">
             <div id="streams_header" class="zoom-in-hide"><h4 class="sidebar-title tippy-zulip-tooltip" data-tippy-content="{{ _('Filter streams') }} (q)">{{ _('STREAMS') }}</h4>
+                {% if page_params.is_web_public_visitor %}
+                <i class="streams_filter_icon web_public" class='fa fa-search tippy-zulip-tooltip' aria-hidden="true" data-tippy-content="{{ _('Filter streams') }} (q)"></i>
```

Comment:
Why do we need to hide this?  If an organization has 50 web-public streams, one might still find this useful...

Reply:
We are not hiding the search button here but the stream settings button which I surely not useful.

Reply:
Ahh, yeah, that's probably reasonable to hide.